start-mongodb.sh

```diff
@@ -51,6 +51,10 @@ echo "  - replica set [$MONGODB_REPLICA_SET]"
 echo ""
 
 docker run --name mongodb --publish $MONGODB_PORT:$MONGODB_PORT --detach mongo:$MONGODB_VERSION --replSet $MONGODB_REPLICA_SET --port $MONGODB_PORT
+if ! docker inspect -f "{{.State.Running}}" "mongodb"; then
+    echo "Docker didn't start."
+    exit 2
+fi
```

Comment:
I think that you have too much thing here instead of only doing what shell has to us, take a look at my suggestion.

```
docker run --name mongodb --publish $MONGODB_PORT:$MONGODB_PORT --detach mongo:$MONGODB_VERSION --replSet $MONGODB_REPLICA_SET --port $MONGODB_PORT
if [ $? -ne 0 ]; then
    echo "Error starting docker"
    exit 2
fi
```

Reply:
Thanks for the hint. I checked it locally. It has the same results.